src/pdfkit.cs

```diff
@@ -448,12 +1206,19 @@ interface PdfDocument : NSCopying {
 		[Export ("writeToFile:withOptions:")]
 		bool Write (string path, NSDictionary options);
 
+		[Wrap ("Write (path, options.Dictionary)")]
+		bool Write (string path, PdfDocumentWriteOptions options);
+
 		[Export ("writeToURL:")]
 		bool Write (NSUrl url);
 
 		[Export ("writeToURL:withOptions:")]
 		bool Write (NSUrl url, NSDictionary options);
 
+		[Wrap ("Write (url, options.Dictionary)")]
```

Comment:
NRE if `options` is `null`
`options?.Dictionary` will give `null` to the overload and that one will throw an ANE